global.json

```diff
@@ -4,7 +4,7 @@
   },
   "sdk": {
     "allowPrerelease": false,
-    "version": "6.0.400",
+    "version": "7.0.100",
```

Comment:
Could you also update https://github.com/Azure/bicep/blob/f1dae160cd495c39ab64dfa889b5bfd7ef8f3f2e/.devcontainer/devcontainer.json#L19

Reply:
Sure! Do I need to do anything special to test it?

Reply:
I don't think it's necessary, but opening Codespaces and ensuring you can run `dotnet build` would be sufficient.

Reply:
Done.